pkg/apis/core/types.go

```diff
@@ -2997,6 +3000,11 @@ type PodSpec struct {
 	// - spec.containers[*].securityContext.runAsGroup
 	// +optional
 	OS *PodOS
+	// ResourceClaims defines which ResourceClaims must be allocated
+	// and reserved before the Pod is allowed to start. The resources
+	// will be made available to those containers which reference them
+	// by name.
```

Comment:
Let's document that this field is immutable.

We are not doing a good job documenting immutability in PodSpec, but let's not make it worse.

Reply:
Agreed and done.